src/main/java/ch/njol/skript/expressions/ExprMessage.java

```diff
@@ -129,6 +117,20 @@ void set(final Event e, final String message) {
 				if (e instanceof PlayerDeathEvent)
 					((PlayerDeathEvent) e).setDeathMessage(message);
 			}
+		},
+		BROADCAST("broadcast", "broadcast(-|[ed] )message", BroadcastMessageEvent.class) {
+			@Override
+			@Nullable String get(Event event) {
+				if (event instanceof BroadcastMessageEvent)
+					return ((BroadcastMessageEvent) event).getMessage();
```

Comment:
Java pattern matching here
```suggestion
				if (event instanceof BroadcastMessageEvent broadcastMessageEvent)
					return broadcastMessageEvent.getMessage();
```